common/app/com/commercetools/sunrise/common/basicauth/BasicAuthHttpFilters.java

```diff
@@ -9,13 +10,13 @@
  * Based on play.mvc.http.DefaultHttpFilters with fix https://github.com/playframework/playframework/pull/6238, which will be released with Play 2.5.5.
  * REMOVE AS SOON AS IT IS RELEASED!
  */
-public class BasicAuthHttpFilters implements HttpFilters {
+public class BasicAuthHttpFilters implements HttpFilters{
```

Comment:
Not the best name now, but since this is a fix and will be fixed soon, I don't want to provide double breaking change.
What I'm not sure now if you can have multiple HttpFilters in your configuration. Have you tried? If not, try to create an additional filter `CsrfHttpFilters` using this one as base. `BasicAuthHttpFilters` will remain as it was before, and `CsrfHttpFilters` will look the same but with the `CSRFFilter` instead. Then you add this one to `application.conf` (be careful with using `+=` instead of `=`.